src/NATS.Client/Info.cs

```diff
@@ -125,7 +131,8 @@ internal StringBuilder AppendAsJsonTo(StringBuilder sb)
                 ["jwt"] = jwt ?? string.Empty,
                 ["nkey"] = nkey ?? string.Empty,
                 ["sig"] = sig ?? string.Empty,
-                ["echo"] = echo
+                ["echo"] = echo,
+                ["headers"] = headers
```

Comment:
To avoid diffs including this line when another property is added:
```suggestion
                ["headers"] = headers,
```